### ISSUE TITLE
Spectre shard not working.

### ISSUE THREAD

**Issue by kindaem** [ValveSoftware/Dota2-Gameplay#12624]
### Hero and spell

Spectre shard Dispersion

### Description

Does spectre supposed to take damage with 100% dispersion?

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/49065330/83761175-30d5-4dad-9aa8-6a6d788cb3fb)

**Comment by LokeLok** [comment]
Yes, since it's a 100% increase from the 8%/12%/16%/20% making it 16%/24%/32%/40% reflect of damage. The tooltip is kinda confusing since people can read it as 100% damage reflect when it's just a doubling to reflected damage.

**Comment by Thinfung91** [comment]
Dota 2 needs a better copywriter
